doc/classes/SceneTree.xml

```diff
@@ -114,36 +114,36 @@
 		<method name="get_node_count" qualifiers="const">
 			<return type="int" />
 			<description>
-				Returns the number of nodes in this [SceneTree].
+				Returns the number of nodes inside this tree.
 			</description>
 		</method>
 		<method name="get_nodes_in_group">
 			<return type="Node[]" />
 			<param index="0" name="group" type="StringName" />
 			<description>
-				Returns a list of all nodes assigned to the given group.
+				Returns an [Array] containing all nodes inside this tree, that have been added to the given [param group].
```

Comment:
You could add that the nodes are sorted with `Node.is_greater_than`, meaning the returned array has a deterministic order.

Reply:
I feel like it's worth to summarise it with "hierarchy order" so long as it refers to the same concept consistently. See updated `Node.is_greater_than()` in https://github.com/godotengine/godot/pull/68560/files.